src/modm/board/nucleo_g071rb/module.lb

```diff
@@ -35,11 +35,6 @@ def build(env):
     env.template("../board.cpp.in", "board.cpp")
     env.copy('.')
 
-    # Waiting on OpenOCD to add support for STM32G0
```

Comment:
Hmmm...
The user still has to compile OpenOCD from source (or get a compiled version of their master branch from somewhere).

Reply:
Yeah but it's not like their latest release v10 is usable either, so you always have to compile from source… Maybe we should start looking at projects? I'm not sure if they are better though.

Reply:
Besides, the STM32G4 BSP has the same issues, but it doesn't even have a warning to remove, so it's already all fubar… 🥺